src/dodal/devices/ophyd_async_xspress3_mini/xspress3_mini_channel_oa.py

```diff
@@ -0,0 +1,28 @@
+from enum import Enum
+
+from ophyd_async.core import Device
+from ophyd_async.epics.signal import epics_signal_r, epics_signal_rw
+
+
+class TimeSeriesValues(str, Enum):
+    START_VALUE = "Acquire"
+    STOP_VALUE = "Done"
+    UPDATE_VALUE = ""
+
+
+class Xspress3MiniChannel(Device):
+    def __init__(self, prefix: str, name: str = "") -> None:
+        # Define some signals
```

Comment:
Could: I'm not sure this comment provides much value. What would be better is a docstring on the device that gives context about what a channel is.